android/app/src/main/AndroidManifest.xml

```diff
@@ -32,6 +32,7 @@
     <uses-permission android:name="android.permission.READ_MEDIA_VIDEO" />
     <uses-permission android:name="android.permission.READ_MEDIA_AUDIO" />
     <uses-permission android:name="android.permission.ACCESS_MEDIA_LOCATION" />
+    <uses-permission android:name="android.permission.POST_NOTIFICATIONS"/>
```

Comment:
please add adr

Reply:
Demo this case, and compare with other application in case of `Don't allow` in android 13

Reply:
https://github.com/linagora/twake-on-matrix/assets/43041967/82e0c7d5-8733-45d1-a271-1417a07eff93